.github/workflows/build-and-deploy.yml

```diff
@@ -7,98 +7,27 @@ on:
-          chmod 600 ~/.ssh/id_rsa
-
-      - name: Add a lando.local file for environment variables.
-        run: |
-          echo "env_file:
-                  - .env  " >> .lando.local.yml
-
-      - name: Add an ephemeral encryption key to the environment variable.
-        run: |
-          echo -e  "ENCRYPTION_PRIVATE_KEY=$(dd if=/dev/urandom bs=32 count=1 | base64 -i -)" >> .env
-
-      - name: Build the Drupal application
-        run: ./oomph-infrastructure/oomph-action build-drupal.yml
-
-      - name: Upload the build archive
-        uses: actions/upload-artifact@v1
-        with:
-          name: build.tar.gz
-          path: /tmp/build.tar.gz
-
-  # Deploy all branches to Acquia
-  branch_deploy:
-    name: Deploy environment branches
+    uses: oomphinc/oomphcloud/.github/workflows/build-drupal.yml@main
+    with:
+      node-version: '10.x'
+      test-drupal: false
+      # For testing, remove once dev/non-dev dependencies are fixed
+      npm-ci-build-options: '--no-fund --no-progress'
+      install-gulp: true
+      drupal-archive-contents: 'config vendor docroot composer.json composer.lock hooks'
+
+  deploy:
     needs: build
-    runs-on: ubuntu-20.04
-    if: startsWith(github.ref, 'refs/heads/')
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Retrieve the infrastructure repo
-        env:
-          INFRA_REPO_DEPLOY_KEY: ${{ secrets.INFRA_REPO_DEPLOY_KEY }}
-        run: ./oomph-actions/retrieve-infra-repo.sh master
-
-      - name: Retrieve the build artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: build.tar.gz
-          path: /tmp/
-
-      - name: Expand the build artifact
-        run: tar -xzf /tmp/build.tar.gz
-
-      - name: Deploy to Acquia
-        env:
-          ACQUIA_SSH_KEY: ${{ secrets.ACQUIA_SSH_KEY }}
-        run: ./oomph-infrastructure/oomph-action deploy-acquia.yml -e "acquia_target_branch=${GITHUB_REF:11}"
-
-  # Deploy tags to Acquia
-  tag_deploy:
-    name: Deploy tags
-    needs: build
-    runs-on: ubuntu-20.04
-    if: startsWith(github.ref, 'refs/tags/')
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-
-      - name: Retrieve the infrastructure repo
-        env:
-          INFRA_REPO_DEPLOY_KEY: ${{ secrets.INFRA_REPO_DEPLOY_KEY }}
-        run: ./oomph-actions/retrieve-infra-repo.sh master
-
-      - name: Retrieve the build artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: build.tar.gz
-          path: /tmp/
-
-      - name: Expand the build artifact
-        run: tar -xzf /tmp/build.tar.gz
-
-      - name: Deploy to Acquia
-        env:
-          ACQUIA_SSH_KEY: ${{ secrets.ACQUIA_SSH_KEY }}
-        run: ./oomph-infrastructure/oomph-action deploy-acquia.yml -e "acquia_target_branch=master"
+    uses: oomphinc/oomphcloud/.github/workflows/deploy-drupal-acquia.yml@main
+    with:
+      acquia-repo-hostname: 'svn-21939.prod.hosting.acquia.com'
```

Comment:
I don't know these 3 variable values...so I temporarily used the values from another project.